mlx_lm/sample_utils.py

```diff
@@ -89,8 +89,11 @@ def logit_bias_processor(_, logits):
         logits_processors.append(logit_bias_processor)
 
     if repetition_penalty and repetition_penalty != 0.0:
+        context_size = (
+            repetition_context_size if repetition_context_size is not None else 20
+        )
         logits_processors.append(
-            make_repetition_penalty(repetition_penalty, repetition_context_size)
+            make_repetition_penalty(repetition_penalty, context_size)
```

Comment:
Revert this change and make `repetition_context_size` not optional. It has a default of 20 which is fine.